lang/en.json

```diff
@@ -371,22 +371,49 @@
 "DND5E.ContextMenuActionEnable": "Enable",
 "DND5E.ContextMenuActionDisable": "Disable",
 "DND5E.ConBlinded": "Blinded",
```

Comment:
As part of this rework I'd suggest maybe moving towards a nested namespace like:
```json
{
  "DND5E": {
    "CONDITIONS": {
      "Blinded": "Blinded",
      "BlindedTooltip": "A blinded creature can't see ..."
     }
   }
}
```
That being said, as related to a separate discussion regarding Rules Pages we may  or may not need BlindedTooltip in the localization.